docs/python_api/datasource/intervals/_intervals.md

```diff
@@ -1,7 +1,25 @@
 ---
-title: Chromatin intevals
+title: Interaction and Interval-based Studies
 ---
 
-# Chromatin intervals
+# List of Interaction and Interval-based Studies
 
-TBC
+In this section, we provide a list of studies that focus on interaction and interval-based investigations, shedding light on the intricate relationships between genetic elements and their functional implications.
```

Comment:
I think this is the right place to add details for the individula interval data sources. My only comment here is that we should provide link to the publications or at least the DOI or pmid.

Reply:
Porvided DOI.